config/fcos/v1_5_exp/translate.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/coreos/butane/config/common"
 	cutil "github.com/coreos/butane/config/util"
 	"github.com/coreos/butane/translate"
+	"gopkg.in/yaml.v3"
```

Comment:
This block is for Butane packages.  External dependencies should be in the next one.